### DISCUSSION TITLE
Add the feature to disable P2P calls to the iOS app

### DISCUSSION THREAD

**Discussion by hyperion-cs** [element-hq/element-meta#1401] [Ideas]
A really common problem is that users cannot hear each other during audio/video calls because Element tries to work in P2P mode. However, because ISPs (especially mobile ISPs) use NAT and other things, it just doesn't work. This problem is a huge headache when using Element, which leaves a very negative user experience from use.
To solve this problem, we [use TURN](https://matrix-org.github.io/synapse/v1.41/turn-howto.html) and it helps. However, in this case we need to disable the use of P2P calls. This can be done in Element Desktop (see screenshot), but there is no such feature in the iOS app.
Thus, I ask very much to add such a feature.

<img width="628" alt="image" src="https://user-images.githubusercontent.com/14217598/230592029-24e9fa3e-96f2-4779-b5ab-9f5e033d0257.png">

_(↑ What the desired feature looks like in Element Desktop)_

**Comment by eNkKVS4HxHc9jaq** [comment]
Certainly I'd appreciate this option in mobile versions too.
